pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -40,11 +40,7 @@
 
 # All/most of these final imports will become obsolete with hdf5 switch
 try:
-    from ligo.lw import utils
-    from ligo.lw.table import Table
     from ligo.segments.utils import fromsegwizard
```

Comment:
Can we get rid of `fromsegwizard` as well ... That one is even older than XML files for segments!!

Reply:
I'll remove the comment.  I was planning to wipe out `fromsegwizard` when I get to the way the segment files are handled in general (i.e., 3 txts vs 1 json file).

Reply:
I also removed the `try/except` to match `pycbc_grb_inj_finder` and `pycbc_grb_trig_combiner`.